CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Grafana Mimir - main / unreleased
 
+### Grafana Mimir
+
+* [ENHANCEMENT] Query-frontend: track all HTTP requests in the Activity Tracker. #3561
```

Comment:
This only tracks query requests, right? (All HTTP requests would include /health or other stuff.)

Reply:
Fair enough, did `s/all/query/` in  https://github.com/grafana/mimir/pull/3561/commits/1a2ecd060719c41fe2a7b99b65c5b37e29ef8313